parmed/charmm/psf.py

```diff
@@ -458,6 +458,10 @@ def load_parameters(self, parmset):
         parmset : :class:`CharmmParameterSet`
             List of all parameters
 
+        copy : bool
+            if False, parmset will not be copied. This will silently modify the references to types in the structure
```

Comment:
I agree with both of you.  It's a perfect opportunity to shoot yourself in the foot.  For instance:

```Python
psf.load_parameters(parmset, copy_parameters=False)

parmset.dihedral_types[('a1', 'a2', 'a3', 'a4')] = DihedralType(1, 2, 3)
```
The above does *not* change the parameter in the `psf` instance.  What `copy_parameters=False` really does is *allow* the paramter set and the structure to share references, but it doesn't enforce it.  So if you change any of the references in the parameter set (i.e., via assignment), then that reference is no longer shared with the structure you loaded the parameters into.  However, the following:
```Python
psf.load_parameters(parmset, copy_parameters=False)

d = parmset.dihedral_types[('a1', 'a2', 'a3', 'a4')]
d.phi_k = 10
d.per = 2
d.phase = 180
```
This *does* change the parameter in `psf` (if it was assigned to one of the torsions, of course).  So this method is very dangerous.  However, it allows someone that knows what they're doing to implement workflows orders of magnitude more efficiently than without it.  That's what ultimately convinced me this was worth the risk.

That said, this needs a better description of what's actually being done here.  @ChayaSt -- can you change the docstring to mention that what this flag really does is to have the PSF share references to types in the parameter set, so if the references are changed in the parameterset, the corresponding parameter in the structure will be changed *and vice versa*.